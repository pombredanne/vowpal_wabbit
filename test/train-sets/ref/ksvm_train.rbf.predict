0.000000
0.154928
0.002690
-0.145346
-0.248784
-0.301046
-0.224803
-0.284798
-0.342736
-0.398398
-0.357676
-0.373931
-0.431610
-0.474000
-0.477409
-0.417575
-0.408166
-0.352343
-0.421632
-0.416758
-0.456949
-0.410170
-0.369701
-0.334157
-0.318735
-0.387099
-0.431610
-0.513994
-0.407565
-0.372802
-0.446135
-0.380476
-0.394165
-0.458369
-0.469026
-0.455446
-0.457908
-0.417736
-0.396344
-0.315591
-0.340630
-0.354105
-0.332440
-0.356648
-0.458097
-0.474740
-0.390370
-0.458497
-0.505620
-0.425879
-0.474391
-0.424684
-0.409130
-0.456089
-0.425197
-0.297122
-0.449238
-0.470502
-0.457967
-0.423558
-0.426483
-0.406630
-0.280845
-0.560059
-0.385793
-0.424778
-0.399779
-0.415577
-0.534908
-0.425221
-0.347228
-0.317971
-0.394165
-0.463931
-0.229299
-0.298650
-0.311975
-0.455124
-0.411396
-0.451468
-0.434632
-0.447860
-0.313664
-0.424896
-0.509575
-0.501285
-0.511938
-0.379045
-0.329397
-0.610239
-0.400488
-0.421543
-0.145396
-0.334470
-0.227824
-0.491528
-0.383960
-0.356979
-0.439350
-0.390568
-0.266970
-0.314236
-0.299810
-0.323606
-0.344313
-0.296612
-0.375367
-0.441414
-0.392871
-0.338148
-0.238853
-0.244516
-0.327912
-0.434927
-0.195123
-0.383585
-0.406667
-0.330260
-0.318942
-0.403725
-0.345072
-0.297523
-0.334778
-0.407420
-0.204083
-0.100252
-0.269347
-0.172401
-0.440566
-0.315246
-0.227098
-0.268202
-0.244852
-0.337733
-0.344875
-0.423334
-0.272021
-0.205421
0.112858
-0.301404
-0.391771
-0.443741
0.001951
0.312000
-0.237941
-0.066696
-0.215802
0.162279
-0.463018
-0.267844
-0.060540
-0.189160
-0.006371
-0.235701
-0.426384
-0.156135
0.244787
-0.204502
0.364158
-0.237503
0.504080
-0.386983
-0.349188
-0.038617
-0.213761
-0.173047
0.245605
-0.268489
-0.255144
0.236940
0.107021
-0.225809
-0.307017
-0.051166
-0.306550
-0.125453
-0.561551
0.160308
-0.070438
0.230641
-0.069315
-0.039515
-0.051989
-0.328259
-0.241040
-0.254936
-0.082877
-0.315019
-0.310483
-0.041049
0.211496
-0.120809
-0.245041
-0.265332
-0.314675
-0.205708
-0.104574
-0.318920
-0.177956
-0.164411
-0.192468
-0.074272
-0.131689
-0.307784
-0.325264
-0.400055
0.036847
-0.131982
-0.242854
-0.131511
-0.194243
-0.402168
-0.331278
-0.155027
0.833763
-0.458134
-0.298019
-0.319512
-0.225733
-0.318702
-0.195697
-0.496075
-0.327622
-0.319066
-0.098696
-0.274090
0.836782
0.050440
-0.298191
-0.407326
-0.468140
-0.526281
-0.489008
-0.226021
-0.172172
-0.471922
-0.433722
0.016524
0.621320
-0.123212
-0.027983
-0.263886
-0.481227
0.269699
-0.210200
-0.080590
0.068184
-0.313019
-0.516401
0.150372
