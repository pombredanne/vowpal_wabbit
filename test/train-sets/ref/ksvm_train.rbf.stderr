using l2 regularization = 1
predictions = ksvm_train.rbf.predict
Lambda = 1
Kernel = rbf
bandwidth = 1
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
using no cache
Reading datafile = train-sets/rcv1_smaller.dat
num sources = 1
average  since         example        example  current  current  current
loss     last          counter         weight    label  predict features
1.000000 1.000000            1            1.0   1.0000   0.0000       50
1.077464 1.154928            2            2.0  -1.0000   0.1549      103
1.003068 0.928672            4            4.0  -1.0000  -0.1453      134
0.944367 0.885665            8            8.0  -1.0000  -0.2848      145
0.929273 0.914179           16           16.0   1.0000  -0.4176       23
0.917699 0.906124           32           32.0  -1.0000  -0.3805       31
0.904987 0.892275           64           64.0  -1.0000  -0.5601       60
0.924562 0.944137          128          128.0   1.0000  -0.1724      105

finished run
number of examples = 250
weighted example sum = 250.000000
weighted label sum = -22.000000
average loss = 0.888309
best constant = -0.088000
best constant's loss = 0.992256
total feature number = 19870
Num support = 250
Number of kernel evaluations = 192975 Number of cache queries = 172366
Total loss = 222.077240
Done freeing model
Done freeing kernel params
Done with finish 
